lib/order.rb

```diff
@@ -0,0 +1,73 @@
+require_relative "customer.rb"
+require "csv"
+
+class Order
+
+  attr_reader :id
+  attr_accessor :customer, :products, :fulfillment_status
+
+  @@orders = []
+  @@found_order = nil
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    raise ArgumentError if ![:pending, :paid, :processing, :shipped, :complete].include? (fulfillment_status)
+
+    @id = id
+    @products = products #give as a hash with name of item and price ("banana" => 1.99)
+    @customer = customer # person who placed the order.  An instance of customer class
+    @fulfillment_status = fulfillment_status
+
+  end
+
+  def total
+    if @products == {}
+      return 0
```

Comment:
if you didn't have this `if`, would the `total` not be `0` for an empty hash `@products`?